sdk/communication/Azure.Communication.AlphaIds/src/Generated/AlphaIdsRestClient.cs

```diff
@@ -105,9 +105,11 @@ internal HttpMessage CreateUpsertConfigurationRequest(bool enabled)
             request.Uri = uri;
             request.Headers.Add("Accept", "application/json");
             request.Headers.Add("Content-Type", "application/merge-patch+json");
-            var model = new AlphaIdConfiguration(enabled);
             var content = new Utf8JsonRequestContent();
-            content.JsonWriter.WriteObjectValue(model);
```

Comment:
Inlined: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/communication/Azure.Communication.AlphaIds/src/Generated/Models/AlphaIdConfiguration.Serialization.cs#L15-L21